### DISCUSSION TITLE
Regarding a Talk (Docker)

### DISCUSSION THREAD

**Discussion by juanpicado** [verdaccio/charts#83] [Ideas]
Hi folks ! @verdaccio/kubernetes @verdaccio/collaborators 

I most probably will give a talk regarding Verdaccio and Docker. Since I know you know more than me. I'm looking for ideas what I might highlight regarding the Docker image (+ integrations), what would you think would be interesting to hear.

My unique ideas I have by far is 

- highlight some examples https://github.com/verdaccio/verdaccio/tree/master/docker-examples
- how to integrate the image with other docker image
- extend the base image for plugins
- how we use github actions for deployment docker images
- what we have learn best practices creating the Docker image over time

So I'd love to hear your opinion here.

**Comment by lirantal** [comment]
I think all of your bullet points are going to amount to a lot of information to share so you probably will already have your hands full with slides for them :-)

A point on DevSecOps and staying secure with regards to supply chain security - in the past we landed a pull request to [detect and prevent secrets from leaking into the repo](https://github.com/verdaccio/verdaccio/blob/a6b0d63afb21c0a0e7eb11435e9924e02f97be1a/website/blog/2019-07-30-verdaccio-420-release.md) which is a good practice and will prevent cases of what happened to Codecov, where an image layer had credentials in it.

**Reply by juanpicado** [reply]
Thanks for the advice :) which remember me I have to restore something 😊 .

**Comment by juanpicado** [comment]
The final title will be  **Using docker and verdaccio to make integration testing easy**

_How many times have you had the need to test publishing one or more Node.js packages to a registry either for local testing or learning?. Verdaccio is a lightweight Node.js proxy registry with an official Docker image with zero-configuration, on this talk, you will learn how to run locally your own private registry and publish Node.js packages, furthermore advance topics with Docker Compose with other images, like Nginx, Apache, LocalStack as S3 local storage._

> I'll also include a small example with Helm v3

 Where: https://www.docker.com/event-community-all-hands

**Reply by deg** [reply]
This sounds excellent; I'm looking forward!
Thank you for taking the time.

**Comment by juanpicado** [comment]
@verdaccio/kubernetes Could you help me to share any particular use case you are using verdaccio in prod? I would like to tell a real story using Docker (even with the Helm chart) and :) only you could give me a real one. 🥺 🥺 🥺 🥺 🥺 🥺  🙏🏼 🙏🏼 🙏🏼 🙏🏼 🙏🏼 🙏🏼
